custom/icds_reports/management/commands/fix_missing_thr_forms.py

```diff
@@ -0,0 +1,134 @@
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+from django.core.management import BaseCommand
+from django.db import connection
+
+from corehq.apps.locations.models import SQLLocation
+from custom.icds_reports.const import (
+    DASHBOARD_DOMAIN
+)
+from custom.icds_reports.tasks import (
+    icds_state_aggregation_task
+)
+query_text_0 = """
+    DROP TABLE IF EXISTS temp_thr
+"""
+
+query_text_1 = """
+    CREATE TEMPORARY TABLE "temp_thr" AS 
+    SELECT
+    awc_location.state_id as state_id,
```

Comment:
is it required to have all these location fields except awc_id which is already present in  thr table